rustfmt.toml

```diff
@@ -1,2 +1,6 @@
 reorder_imports = true
 reorder_imported_names = true
+
+wrap_comments = false
```

Comment:
I wanted to prevent rustfmt to mangle commented out code. But since this is the default, I'll remove it.